README.md

```diff
@@ -1,112 +1,207 @@
-   and the PR page shows the final merged commit id.
+3. Each commit on the hotfix branch references a PR, and that PR was
+   reviewed, and the PR page shows the final merged commit id.
 
-## Example
-
-* On `2018-01-30`, @cpu submits [a pull request](https://github.com/letsencrypt/boulder-release-process/pull/1), proposing a feature to be merged to `master`.
+## Prerequisites
 
-* @jsha reviews and [approves the PR](https://github.com/letsencrypt/boulder-release-process/pull/1#pullrequestreview-92717219). The corresponding [travis build](https://travis-ci.com/letsencrypt/boulder-release-process/builds/64669844?utm_source=github_status&utm_medium=notification) passes, and so [@cpu merges the PR](https://github.com/letsencrypt/boulder-release-process/commit/500b4c3c010b7f8d97aca106aa76550b5cff72f0) into master.
+* You must have a GPG key with signing capability:
+  * [Checking for existing GPG
+    keys](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/checking-for-existing-gpg-keys)
 
-* Later that day, @cpu cuts a release by making a [`release-2018-01-30` tag](https://github.com/letsencrypt/boulder-release-process/releases/tag/release-2018-01-30) from the [tip of `master`](https://github.com/letsencrypt/boulder-release-process/commit/500b4c3c010b7f8d97aca106aa76550b5cff72f0) at the time of tagging.
-
-* Ops deploys `release-2018-01-30` some time later.
+* If you don't have a GPG key with signing capability, create one:
+    * [Generating a new local GPG
+      key](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/generating-a-new-gpg-key)
+    * [Generating a new Yubikey GPG
+      key](https://support.yubico.com/hc/en-us/articles/360013790259-Using-Your-YubiKey-with-OpenPGP)
 
-* After deploying `release-2018-01-30` a critical bug is discovered! Since `master` is clean, that is, has no commits since `release-2018-01-30` we don't want to ship with a hotfix, @cpu quickly prepares a [hotfix PR](https://github.com/letsencrypt/boulder-release-process/pull/2) to propose a hotfix change be merged into `master`.
+* The signing GPG key must be added to your GitHub account:
+  * [Adding a new GPG key to your GitHub
+    account](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/adding-a-new-gpg-key-to-your-github-account)
 
-* @jsha reviews and [approves the hotfix PR](https://github.com/letsencrypt/boulder-release-process/pull/2#pullrequestreview-92737652). The corresponding [travis build](https://travis-ci.com/letsencrypt/boulder-release-process/builds/64675859?utm_source=github_status&utm_medium=notification) passes, and so [@cpu merges the hotfix PR](https://github.com/letsencrypt/boulder-release-process/commit/4ed5f8181852237dd6cde45fbc63bd199f2d3ec9) into master.
+* `git` must be configured to call the correct GPG binary:
+  * On most Linux platforms: `git config --global gpg.program gpg`
```

Comment:
I believe this is the default, and isn't necessary most of the time.

Reply:
I had somehow set mine to gpg2 and that was not right, I don't want someone else to experience the same pain of figuring that out if it can be helped.

Reply:
I have reworded this. If it still doesn't make sense to keep it, I can remove it.